auth_admin_passkey/model/res_users.py

```diff
@@ -94,20 +79,20 @@ def _send_email_same_password(self, cr, login_user):
     def authenticate(self, db, login, password, user_agent_env):
         """ Authenticate the user 'login' is password is ok or if
  is admin password. In the second case, send mail to user and admin."""
-        user_id = super(res_users, self).authenticate(
+        user_id = super(ResUsers, self).authenticate(
             db, login, password, user_agent_env)
         if user_id and (user_id != SUPERUSER_ID):
             same_password = False
-            cr = pooler.get_db(db).cursor()
+            cr = registry(db).cursor()
```

Comment:
I think `self.pool.cursor()` is the way to get the cursor. I don't see any examples of this way in v9 code